### ISSUE TITLE
Additional features inspired by Windows clipboard manager

### ISSUE THREAD

**Issue by MrTomRod** [SUPERCILEX/gnome-clipboard-history#136]
### Problem description

Having had to spend a few weeks on Windows 🤮, I noticed a few niceties they built into their clipboard manager:

- If opened using keyboard shortcut, the window appears close to the cursor.
- Image previews
- Emoji selector
- Special characters selector

### Potential solutions/workarounds

This is what their manager looks like:

![image](https://www.popsci.com/uploads/2022/05/19/Windows-Clipboard-History.jpeg)

### Additional context

These are just ideas. I don't expect anyone to implement them for me, just trying to communicate the idea.

**Comment by Kezzers** [comment]
Leaving a comment to say I have no idea what I'm doing, but just got Linux installed on my Laptop. Loving this extention! Just wish it also saved Pictures copied (either through Screenshot - saved to clipboard, or copying images)

**Comment by santropedro** [comment]
> If opened using keyboard shortcut, the window appears close to the cursor.
I agree this would be a good feature, but not crucial by any means, only saves maybe 0.5 seconds.
> Emoji selector
Totally unnecessary, unicode has emojis https://unicode.org/emoji/charts/full-emoji-list.html anyway so it should be part of inserting special characters.
> Image previews
We can't live without that if it's too complicated, one normally copies one image at a time.

> Special characters selector
That could be a feature, however it's complex, maybe there are another tools to do that. I do that with the text editor or by googling the character's description. So it seems way too complex, I would recommend abstaining from this.
